bridge/device/cloud/signUp.go

```diff
@@ -80,9 +80,7 @@ func (c *Manager) signUp(ctx context.Context) error {
 	if err != nil {
 		return errCannotSignUp(err)
 	}
-	if signUpResp.ExpiresIn != -1 {
-		c.creds.ValidUntil = validUntil(signUpResp.ExpiresIn)
-	}
+	signUpResp.ValidUntil = validUntil(signUpResp.ExpiresIn)
```

Comment:
Is this correct? Previously `signUpResp.ValidUntil` retained it's value when `ExpiresIn==-1`, now it will be set to `time.Time{}` by `validUntil`

Reply:
This is correct - it is same as before(not set).